VTEX - MasterData API - v10.2.json

```diff
@@ -967,820 +717,893 @@
                     "required": true
                 },
                 "responses": {
-                    "200": {
-                        "description": "",
-                        "headers": {}
+                    "204": {
+                        "description": "No Content"
                     }
                 },
                 "deprecated": false
             },
             "patch": {
-                "tags": ["Documents"],
+                "tags": [
+                    "Documents"
+                ],
                 "summary": "Update partial document",
-                "description": "This endpoint can be used to update a subset of fields of a document, without impacting the other fields.",
+                "description": "Updates a subset of fields of a document, without impacting the other fields.\r\n\r\n> You can use this request to update documents in any given data entity. Because of this, you are not restricted to using the fields exemplified below in your requests. But you should be aware of the fields allowed or required for each document you wish to update.",
```

Comment:
```suggestion
                "description": "Updates a subset of fields of a document, without impacting the other fields.\r\n\r\n>ℹ️ You can use this request to update documents in any given data entity. Because of this, you are not restricted to using the fields exemplified below in your requests. But you should be aware of the fields allowed or required for each document you wish to update.",
```